app/controllers/distributions_controller.rb

```diff
@@ -7,6 +7,7 @@ class DistributionsController < ApplicationController
   include DateRangeHelper
   include DistributionHelper
 
+  before_action :enable_turbo!, only: %i[index]
```

Comment:
Is there a reason we're only enabling turbo on one page?

Reply:
Turbo breaks a lot of pages (specifically flash messages) because it changes the content type to TURBO_STREAM. This article is pretty good at explaining moving to Hotwire with legacy JS - https://dev.to/nejremeslnici/using-hotwire-turbo-in-rails-with-legacy-javascript-17g1

Reply:
But this is Turbo Drive, not Turbo Stream, correct? They are two different things and enabling one shouldn't affect the other.

Reply:
Turbo Drive is a library that transforms all links and interactions into content-type TURBO_STREAM. 
> Turbo Drive is the part of Turbo that enhances page-level navigation. It watches for link clicks and form submissions, performs them in the background, and updates the page without doing a full reload. It’s the evolution of a library previously known as [Turbolinks](https://github.com/turbolinks/turbolinks).

Without this specificity, almost all system specs break because the flash messages don't render if the page doesn't refresh due to turbo just replacing the body.

New territory for me as well!

Reply:
There is definitely a way to fix this so that flash messages are displayed correctly even with Turbo rather than disabling it entirely. Here's one tutorial: https://www.hotrails.dev/turbo-rails/flash-messages-hotwire

Reply:
@dorner yep. I did manage to fix the flash messages in a commit that I didn't push yet. However, it does require going into each controller to handle it from what I can tell. At a high level, we need to handle the TURBO_STREAM content and either does a `replace` or `append` to the flash partial on the layout to get the same effect.

I'll see if I can push that up later today and demonstrate it working.

Reply:
Got it work in - https://github.com/rubyforgood/human-essentials/pull/3119/commits/cff84460e99b18cab45cbb3cf6957403787ab1a1; This requires a bit of work to get it to work. I'll make sure to add issues to start widely supporting turbo so we don't need to specifically say which parts can use turbo or not.